### ISSUE TITLE
.delete() does not exist on an item retrieved by .get()

### ISSUE THREAD

**Issue by GordyD** [neumino/thinky#238]
Executing the following code:

```
return that.Model.get(id)
  .then(function(model) {
    return model.delete();
  });
```

Results in:

```
[TypeError: Object #<Object> has no method 'delete']
```

I have a workaround currently working but I'd appreciate clarification on why this is the case as the [documentation](https://thinky.io/documentation/api/document/#delete) explicitly states that what I am attempting is expected behaviour.

**Comment by neumino** [comment]
Try to add run after get. There's a PR for that by Simon. I'll merge it
tonight or tomorrow (tonight if I can get home at a reasonable time)

**Comment by GordyD** [comment]
I just noticed this myself! Beat me to it. +1 for incredibly quick response.

**Comment by neumino** [comment]
https://github.com/neumino/thinky/pull/222 was merged yesterday.
It's fixed in 2.0.3
